Sources/Kitura/RouterHTTPVerbs_generated.swift

```diff
@@ -28,7 +28,7 @@ extension Router {
     /// - Parameter handler: A comma delimited set of `RouterHandler`s that will be
```

Comment:
The code in this file should *not* be modified. It is generated code, as it's name implies

Please fix the generator in `Scripts/generate_router_verbs.sh`, run the code and push the generated code